lib/rails_erd/domain/attribute.rb

```diff
@@ -88,6 +88,11 @@ def inheritance?
         @model.inheritance_column == name
       end
 
+      # Method allows false to be set as an attributes option when making custom graphs.
+      # Its functionality is implemented in the filtered_attributes method.
+      def false?
+      end
```

Comment:
Do you not know what these methods are for or why tests require them?

On Mon, Oct 26, 2015 at 10:00 AM, Kerri Miller <notifications@github.com>
wrote:

> In lib/rails_erd/domain/attribute.rb
> <https://github.com/voormedia/rails-erd/pull/173#discussion_r43020923>:
>
> > @@ -88,6 +88,11 @@ def inheritance?
> >          @model.inheritance_column == name
> >        end
> >
> > +      # Method allows false to be set as an attributes option when making custom graphs.
> > +      # Its functionality is implemented in the filtered_attributes method.
> > +      def false?
> > +      end
>
> My concern is that we shouldn't add a method that we don't understand the
> purpose of. Maybe we can mob on this sometime this week and trace it down?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/voormedia/rails-erd/pull/173/files#r43020923>.
>